guides/source/active_model_basics.md

```diff
@@ -65,7 +65,7 @@ irb> person.age_highest?
 ### Callbacks
 
 `ActiveModel::Callbacks` gives Active Record style callbacks. This provides an
-ability to define callbacks which run at appropriate times.
+ability to define callbacks that run at appropriate times.
```

Comment:
Although I prefer `that` here instead of `which`, `which` seems still allowed according to Merriam-Webster:

> if your clause is bracketed by commas (“the article on grammar, which I started while eating lunch, seemed to never end”) it is likely a nonrestrictive clause, and you can give it a which. If it is not surrounded by commas, then it is most likely a restrictive clause, and you can choose to give it a that or a which. If anyone questions your decision, you can say that you are following the advice of the Fowlers, and are making a decision based on custom, euphony, and convenience. 

https://www.merriam-webster.com/words-at-play/when-to-use-that-and-which

This change is arbitrary and someone else might just as well make the case to change it back to `which`.